src/prg_quantumdynamics_mod.F90

```diff
@@ -300,12 +220,15 @@ end subroutine prg_kick_density_bml
   !! \param A matrix in first index of the commutator
   !! \param B matrix in second index of the commutator
   !! \param C output matrix that is the commutator of A and B
-  subroutine prg_commutate_bml(A,B,C)
+  !! \param thresh the BML threshold
+  !!
+  subroutine prg_commutate_bml(A,B,C,thresh)
```

Comment:
@cnegre Maybe we should think about adding that to the bml itself. What do you think?